t/app/controller/waste_bexley.t

```diff
@@ -477,6 +477,32 @@ FixMyStreet::override_config {
         $mech->content_contains('<a href="/report/' . $existing_missed_collection_report2->id . '" class="waste-service-link">check status</a>');
     };
 
+    subtest 'Allows missed collection reports if within 3 days of round completion' => sub {
+        $mech->get_ok('/waste/10001');
+        $mech->content_contains('Report a clear sack(s) collection as missed');
+    };
+
+    subtest 'Prevents missed collection reports if there is no round logs and 4 days or more after collection date' => sub {
+        $whitespace_mock->mock( 'GetInCabLogsByUprn', sub {
+            my ( $self, $uprn ) = @_;
+            return _empty_cab_logs()->{$uprn};
+            });
```

Comment:
Is there any advantage to having the `_empty_cab_logs` method and passing `$uprn` around, rather than simply returning an empty arrayref?


```
$whitespace_mock->mock( 'GetInCabLogsByUprn', sub { [] } );
```

Reply:
Same here :-)